fastlane/Fastfile

```diff
@@ -246,6 +246,11 @@ platform :ios do
         './Tests/BackendIntegrationTests/Constants.swift',
         './Tests/InstallationTests/CommonFiles/RCInstallationRunner.m'
       ]
+      previous_text: "REVENUECAT_LOAD_SHEDDER_API_KEY",
+      new_text: ENV["REVENUECAT_LOAD_SHEDDER_API_KEY"],
```

Comment:
Need to set this in CircleCI once it's available.

Reply:
Hmm I think you will need a new call to `replace_text_in_files` here for this replacement?

Reply:
What am I doing 🤦🏻